foundation/foundation-mda/src/main/resources/templates/cucumber.spark.test.base.harness.java.vm

```diff
@@ -34,7 +34,7 @@ public abstract class SparkTestBaseHarness implements EventListener {
 
 	private static final Logger logger = LoggerFactory.getLogger(SparkTestBaseHarness.class);
 	protected static final SparkConfig config = KrauseningConfigFactory.create(SparkConfig.class);
-    private static final File DEFAULT_SPARK_TEST_APPLICATION_SPEC = new File("./target/apps/${artifactId}-test-chart.yaml");
+    private static final File DEFAULT_SPARK_TEST_APPLICATION_SPEC = new File("./target/apps/aissemble-spark-application-chart/templates/deployment.yaml");
 #if (${pipeline.isHiveSupportNeeded()})
```

Comment:
C: I don't think this is correct when a downstream project is being created. We generate the application yaml files based on the pipeline name. This will break downstream project pipeline tests.

Reply:
will update to use exec-maven-plugin and revert back to original!